### PR TITLE
FEAT: added Venture Labs brand banner

### PR DESCRIPTION
**Pull request by 23po**
# Description

Created Ventures Labs brand banner on brands page.

To achieve this,:

```
- I customized the reusable brands banner
```

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)

# Screenshot (optional)
venture labs

# How Has This Been Tested?

This runs on the development server

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by 23po** [comment]
@IanOdhiambo9 please review

File: libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts [hunk 1]
```diff
@@ -9,9 +9,14 @@ import { ElewaBrandsOpportunitiesComponent } from './components/elewa-brands-opp
 import { ElewaBrandsPageComponent } from './pages/elewa-brands-page/elewa-brands-page/elewa-brands-page.component';
 
 import { BrandsRoutingModule } from './brands.routing';
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/brands/src/lib/pages-elewa-brands.module.ts:11]: order of imports

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]
resolve conflicts

**Comment by 23po** [comment]
@IanOdhiambo9 resolved

**Comment by IanOdhiambo9** [comment]
merged one of the same

**Pull request closed without merging**